pkg/cli/run.go

```diff
@@ -88,14 +88,17 @@ type ScanOptions struct {
 	// severity is the level of severity of policy violations that should be reported
 	severity string
 
-	// verbose indicates whether to display all fields in default human readlbe output
+	// verbose indicates whether to display all fields in default human readable output
 	verbose bool
 
 	// showPassedRules indicates whether to display passed rules or not
 	showPassedRules bool
 
 	// nonRecursive enables recursive scan for the terraform iac provider
 	nonRecursive bool
+
+	// useTerrafomeCache provides ability to use terraform init local cache for modules rather than downloading them.
+	useTerrafomeCache bool
```

Comment:
Should this be `useTerraformCache` for consistency?

Reply:
yes this was a typo, I have fixed this